careers/index.md

```diff
@@ -24,20 +24,42 @@ We are seeking a dedicated and skilled Field Engineer to join our team. In this
 - Maintain accurate records of activities, including test cases, results, and observations.
 - Collaborate with cross-functional teams to ensure alignment of testing and demonstration efforts with project timelines and objectives.
 
+
 ### Qualifications
-- Bachelor's degree in Computer/Electrical Engineering, Computer Science, or a related field. Fresh graduates are welcome to apply.
-- Experience in basic software engineering to troubleshoot issues and develop simple test scripts.
-- Demonstrated experience in assisting Project and Product Managers with organizing and executing field activities and creating and implementing testing plans.
-- Strong analytical skills with attention to detail.
-- Effective communication and interpersonal skills, with the ability to work collaboratively in a team environment.
-- Willingness to travel and work in various field locations as required including on boats in local waters of Singapore.
-- Knowledge of relevant industry standards and regulations is a plus.
-- Ability to work independently across multiple time zones is a must.
-- Comfortable operating within a fast-paced environment.
+- Bachelor’s degree in Electrical Engineering, Mechanical Engineering, Ocean Engineering, or a related field.
+- Proven experience in subsea engineering, IoT deployment, or underwater robotics, preferably in an offshore or marine environment.
+- Experience in subsea system communication protocols, such as acoustic modems, RF communication, and satellite communication is preferred.
```

Comment:
```suggestion
- Experience with communication protocols, such as acoustic modems, RF communication, or satellite communication will be valuable.
```

Otherwise this is gonna be hard to find anyone.